src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -310,24 +310,34 @@ public partial class UpdateDurableFunction
         }
     }
 
-    private IReadOnlyList<InstanceMetadata> GetInstanceMetadataList(IReadOnlyList<InstanceMetadata> instanceMetadataList)
+    private async Task HandleException(
+        IDurableOrchestrationContext context,
+        UpdateCheckpoint input,
+        string studyInstanceUid,
+        IReadOnlyList<InstanceMetadata> instances,
+        IReadOnlyList<string> instanceErrors)
     {
-        // when external store not enabled, do not update file properties
-        return _externalStoreEnabled ? instanceMetadataList : new List<InstanceMetadata>();
+        var errors = new List<string>
+        {
+            $"Failed to update instances for study {studyInstanceUid}",
+        };
+
+        if (instanceErrors != null)
+            errors.AddRange(instanceErrors);
+
+        if (input.Errors != null)
+            errors.AddRange(errors);
+
+        input.Errors = errors;
```

Comment:
So does that mean the output could have a giant list of strings detailing the errors?

Reply:
This seems more like a job for shoebox

Reply:
Sure, for giant list, we do the same in STOW for a multipart request. If there is a validation errors, we tell the user about the list of validation errors

Reply:
Discussed during our meeting and also with synced with Matt, We will send the failure both to the user as well as to shoebox.

Reply:
Is there a max size for storing the data in the table storage?

Reply:
I do not recall anything, and the DTFx may attempt to compress the entry as needed using `GZip` and storing in blob storage. But I think we should also be wary of what the user experience is for this via the response. Should the response be potentially huge with a lot of strings? Instead, should the response just be a summary like "5 instances failed to update" with the logs having more details?

Reply:
Make sense, I will add upto 5 errors and add more details.